internal/model/config.go

```diff
@@ -34,3 +35,14 @@ type ErrorLog struct {
 	Permissions string
 	Readable    bool
 }
+
+type (
```

Comment:
If `WriteStatus` is only used in the `file` package, then I think it makes sense to move this enum there instead of the `model` package.

Reply:
Was talking with Oliver about it, had an issue with import cycle and we might end up using it for the status later when we implement it.